src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -37,227 +114,2413 @@ const ActorInit En_Ossan_InitVars = {
+                   G_TX_NOMASK, G_TX_NOLOD, G_TX_NOMIRROR | G_TX_WRAP, 4, G_TX_NOLOD);
+        gDPSetTileSize(OVERLAY_DISP++, G_TX_RENDERTILE, 0, 0, 15 * 4, 15 * 4);
+        if (drawStickRightPrompt) {
+            EnOssan_DrawTextRec(globalCtx, this->stickRightPrompt.stickColor.r, this->stickRightPrompt.stickColor.g,
+                                this->stickRightPrompt.stickColor.b, this->stickRightPrompt.stickColor.a,
+                                this->stickRightPrompt.stickTexX, this->stickRightPrompt.stickTexY,
+                                this->stickRightPrompt.z, 0, 0, -1.0f, 1.0f);
+        }
+        if (drawStickLeftPrompt) {
+            EnOssan_DrawTextRec(globalCtx, this->stickLeftPrompt.stickColor.r, this->stickLeftPrompt.stickColor.g,
+                                this->stickLeftPrompt.stickColor.b, this->stickLeftPrompt.stickColor.a,
+                                this->stickLeftPrompt.stickTexX, this->stickLeftPrompt.stickTexY,
+                                this->stickLeftPrompt.z, 0, 0, 1.0f, 1.0f);
+        }
+    }
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_oB1.c", 4300);
+}
+
+void EnOssan_DrawBazaarShopkeeper(Actor* thisx, GlobalContext* globalCtx) {
+    static u64* sBazaarShopkeeperEyeTextures[] = { gOssanEyeOpenTex, gOssanEyeHalfTex, gOssanEyeShutText };
+    EnOssan* this = THIS;
+    s32 pad;
+
+    OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_oB1.c", 4320);
+
+    func_80093D18(globalCtx->state.gfxCtx);
+    gSPSegment(POLY_OPA_DISP++, 0x08, SEGMENTED_TO_VIRTUAL(sBazaarShopkeeperEyeTextures[this->eyeTextureIdx]));
+    SkelAnime_DrawFlexOpa(globalCtx, this->skelAnime.skeleton, this->skelAnime.jointTable, this->skelAnime.dListCount,
+                          EnOssan_OverrideLimbDrawDefaultShopkeeper, NULL, this);
+    EnOssan_DrawCursor(globalCtx, this, this->cursorX, this->cursorY, this->cursorZ, this->drawCursor);
+    EnOssan_DrawStickDirectionPrompts(globalCtx, this);
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_oB1.c", 4340);
+}
+
+s32 EnOssan_OverrideLimbDrawKokiriShopkeeper(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos,
+                                             Vec3s* rot, void* thisx) {
+    static u64* sKokiriShopkeeperEyeTextures[] = { 0x06001570, 0x060001F0, 0x06000B30 };
+    EnOssan* this = THIS;
+    s32 pad;
+
+    OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_oB1.c", 4354);
+
+    if (limbIndex == 15) {
+        gSPSegment(POLY_OPA_DISP++, 0x06, globalCtx->objectCtx.status[this->objBankIndex2].segment);
+        gSegments[6] = VIRTUAL_TO_PHYSICAL(globalCtx->objectCtx.status[this->objBankIndex2].segment);
+        *dList = D_06002820;
+        gSPSegment(POLY_OPA_DISP++, 0x0A, SEGMENTED_TO_VIRTUAL(sKokiriShopkeeperEyeTextures[this->eyeTextureIdx]));
+    }
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_oB1.c", 4374);
+
+    return 0;
+}
+
+// Graph_Alloc EndDisplayList
+Gfx* func_80AC801C(GraphicsContext* gfxCtx) {
```

Comment:
```suggestion
Gfx* EnOssan_DListEnd(GraphicsContext* gfxCtx) {
```
Or DlEnd

Reply:
at a higher level its purpose is probably to stub a dlist in the case they dont want to add anything extra to a dynamic section of a dlist. not sure if we need to literally say the name of the command in the name